src/game/lanes_and_cars/pathfinding/mod.rs

```diff
@@ -248,19 +248,20 @@ pub fn setup(system: &mut ActorSystem) {
                 })
                 .unwrap_or(true);
             if join {
+                let tell_to_forget_next_tick = lane.pathfinding
```

Comment:
had to rearrange as it failed with borrowing errors I did not have time to dig into